src/partials/footer.hbs

```diff
@@ -14,13 +14,13 @@
       <span id="copyrightdate">{{year}}</span>
       <span>DataStax, an IBM Company |
         <a
-          href="https://www.datastax.com/legal/datastax-website-privacy-policy"
+          href="https://www.ibm.com/us-en/privacy"
```

Comment:
```suggestion
          href="https://www.ibm.com/privacy"
```

I think this should auto-resolve to the correct geo

Reply:
Great catch!